optimade/server/schemas.py

```diff
@@ -58,11 +58,17 @@ def retrieve_queryable_properties(
             else:
                 properties[name] = {"description": value.get("description", "")}
                 # Update schema with extension keys provided they are not None
-                for key in [_ for _ in ("unit", "queryable", "support") if _ in value]:
-                    properties[name][key] = value[key]
+                for key in (
+                    "x-optimade-unit",
+                    "x-optimade-queryable",
+                    "x-optimade-support",
+                    "x-optimade-unit",
```

Comment:
You have listed "x-optimade-unit" unit twice here. That seems a bit strange.

Reply:
Hmm yes, this is a typo, though not a consequential one. I will remember to fix it in the next PR I make